CodeReview/Amie_code.py

```diff
@@ -9,9 +9,12 @@
 %matplotlib inline
 #Load data
 data = pd.read_csv('../Data/2014_PortTownsend.csv', parse_dates=['Date Time'])
+
 import plotly.plotly as py
 import plotly.graph_objs as go
+#Use date time index
 data_datetime = pd.DatetimeIndex(data['Date Time'])
+#Edit data adding new columns
 def f(data):
     data = data.copy()
```

Comment:
In here, I actually think that you don't need to copy the data frame, the original data variable wont be modified within the function. When you call the function you can say data_modified = f(data), since f returns the modified data dataframe